apache-rat-core/src/main/java/org/apache/rat/configuration/XMLConfig.java

```diff
@@ -23,36 +23,36 @@
 /**
  * Configuration definitions for XMLConfiguration reader and writer.
  */
-public class XMLConfig {
+public final class XMLConfig {
 
     /** id attribute name */
-    public final static String ATT_ID = "id";
+    public static final String ATT_ID = "id";
     /** name attribute name */
-    public final static String ATT_NAME = "name";
+    public static final String ATT_NAME = "name";
     /** license reference attribute name */
-    public final static String ATT_LICENSE_REF = "license_ref";
+    public static final String ATT_LICENSE_REF = "license_ref";
     /** class name attribute name */
-    public final static String ATT_CLASS_NAME = "class";
+    public static final String ATT_CLASS_NAME = "class";
     /** resource file name attribute name. */
-    public final static String ATT_RESOURCE = "resource";
+    public static final String ATT_RESOURCE = "resource";
     /** root of the configuration file */
-    public final static String ROOT = "rat-config";
+    public static final String ROOT = "rat-config";
     /** families element name */
-    public final static String FAMILIES = "families";
+    public static final String FAMILIES = "families";
     /** licenses element name */
-    public final static String LICENSES = "licenses";
+    public static final String LICENSES = "licenses";
     /** license element name */
-    public final static String LICENSE = "license";
+    public static final String LICENSE = "license";
     /** approved element name */
-    public final static String APPROVED = "approved";
+    public static final String APPROVED = "approved";
     /** family element name */
-    public final static String FAMILY = "family";
+    public static final String FAMILY = "family";
     /** note element name */
-    public final static String NOTE = "note";
+    public static final String NOTE = "note";
```

Comment:
This constant is unused - does it get called via reflection or can we safely remove it here? @Claudenw